bin/node/runtime/src/lib.rs

```diff
@@ -1534,6 +1574,9 @@ impl pallet_uniques::Config for Runtime {
 	type CollectionId = u32;
 	type ItemId = u32;
 	type Currency = Balances;
+	type CurrencyBalance = <Self as pallet_balances::Config>::Balance;
+	type Assets = Assets;
+	type AssetId = <Self as pallet_assets::Config<Instance1>>::AssetId;
```

Comment:
Why the change to the `pallet_uniques` config?

Reply:
Sorry this should have been reverted. Child PR to fix that here: https://github.com/paritytech/substrate/pull/12991

Reply:
Fixed